st2common/st2common/rbac/types.py

```diff
@@ -91,6 +94,7 @@ class PermissionType(Enum):
     EXECUTION_RE_RUN = 'execution_rerun'
     EXECUTION_STOP = 'execution_stop'
     EXECUTION_ALL = 'execution_all'
+    EXECUTION_VIEWS_FILTERS_LIST = 'execution_view_filters_list'
```

Comment:
Yeah, was thinking about it quite a bit, but I prefer to be explicit :P

"pack health" vs "pack index health" - first one is a bit confusing and not really clear, imo.